system/ee/ExpressionEngine/Model/Member/Member.php

```diff
@@ -481,6 +503,13 @@ public function onBeforeDelete()
 
     public static function onAfterBulkDelete()
     {
+        if (ee()->config->item('ignore_member_stats') != 'y') {
+            foreach (ee('Model')->get('Role')->all() as $role) {
+                $role->total_members = null;
+                $role->save();
```

Comment:
@intoeetive does this already basically run from the onAfterDelete (assuming that gets run for each member?  Or does it get skipped in bulk delete?)

Reply:
Your are correct. Removed